homeassistant/components/wemo.py

```diff
@@ -14,19 +14,20 @@
 
 from homeassistant.const import EVENT_HOMEASSISTANT_STOP
 
-REQUIREMENTS = ['pywemo==0.4.20']
+REQUIREMENTS = ['pywemo==0.4.22']
 
 DOMAIN = 'wemo'
 
 # Mapping from Wemo model_name to component.
 WEMO_MODEL_DISPATCH = {
-    'Bridge':  'light',
     'Insight': 'switch',
     'Maker':   'switch',
     'Sensor':  'binary_sensor',
     'Socket':  'switch',
     'LightSwitch': 'switch',
-    'CoffeeMaker': 'switch'
+    'Dimmer': 'light',
+    'CoffeeMaker': 'switch',
```

Comment:
Decide on one way to sort this. Either have the keys sorted 🔤 or group the components. Or don't change the sorting and just append to the end.

Reply:
Cleaning this up right now on my dev machine, will publish the update after I have tested it